expenses/migrations/0001_initial.py

```diff
@@ -11,7 +11,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('house', '0002_add_initial_data'),
```

Comment:
why did you change that line?

Reply:
It happened alone when I was running "run migrations"
I need to change it? @ronyyosef